src/main/java/seedu/address/model/Model.java

```diff
@@ -59,31 +61,64 @@ public interface Model {
      */
     boolean hasPerson(Person person);
 
+    /**
+     * Returns true if the same company is in the address book.
+     *
+     * @param company Company to be checked.
+     * @return true if in address book.
+     */
+    boolean hasCompany(Company company);
+
```

Comment:
For the JavaDocs in general, are we sticking to the existing code base style, or using this style?